config/quickwit.yaml

```diff
@@ -20,20 +20,23 @@ version: 0
 # - for its gRPC service (TCP)
 # - for its Gossip cluster membership service (UDP)
 #
-# All three services are bound to the same host and different port.
+# All three services are bound to the same host and a different port. The host can be an IP address or a hostname.
 #
 # Default HTTP server host is `127.0.0.1` and default HTTP port is 7280.
```

Comment:
For good reasons, our default is not a sensible value for most use case.

Can you add a comment explaining that `127.0.0.1` cannot work when working on more than one node,
and document the behavior of `0.0.0.0`

Reply:
Will do!